umbra/umbra_64.c

```diff
@@ -354,6 +358,37 @@ static ptr_uint_t map_disp_win81[] = {
  * We check conflicts later when creating shadow memory.
  */
 #ifdef LINUX
+#ifdef AARCH64
+static const app_segment_t app_segments_initial[] = {
+    /* We split app3 [0x7F0000000000, 0x800000000000) into two parts:
```

Comment:
I see that a big chunk of this comment block was copied from that below. Consider condensing to avoid duplicate text, as future updates would then require double changes.

Reply:
Removed comment.